python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -156,3 +154,68 @@ def compile(model: nn.Module,
                 return pl_model
         else:
             return pl_model
+
+    def quantize(self, pl_model, calib_dataloader, val_dataloader=None, metric: Metric = None,
+                 backend='inc', conf=None, framework='pytorch_fx', approach='static',
```

Comment:
why only `metric` has typing hints?

Reply:
Some of them has default values, so the type hint is not needed. I'll add hint for `conf `and `accuracy_criterion`

Reply:
Will also add type hint for pl_model, calib_dataloader, val_dataloader.

Reply:
After modification:
![image](https://user-images.githubusercontent.com/90437536/146733355-b8709e47-9d75-45cb-bb27-ef8043222738.png)